docs/section19/19-9.md

```diff
@@ -0,0 +1,34 @@
+# Section 19 - Rule 19-9     
+**Schema Version:** 0.0.22  
+**Mandatory Rule:** True    
+**Rule ID:** 19-9     
+**Rule Description:** Air economizers shall not be included in baseline HVAC Systems 1, 2, 9, and 10.    
+**Rule Assertion:** Options are Pass/Fail/NOT_APPLICABLE     
+**Appendix G Section:** G3.1.2.6      
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Each HeatingVentilatingAirConditioningSystem  
+
+**Applicability Checks:** 
+1. Applies to baseline HVAC system types 1, 2, 9, and 10.
+
+**Function Calls:**  
+1. get_baseline_system_types  
+
+## Rule Logic:   
+**Applicability Check 1**   
+- Create ditionary of baseline system types: `baseline_system_types_dict = get_baseline_system_types(B_RMR)`  
+- Check if B-RMR is modeled with at least one air-side system that is Type-1,1a,1b,1c,2,9,9b,10, if yes then carry on, if no then NOT_APPLICABLE for the project: `for sys_type in ["SYS-1", "SYS-1a", "SYS-1b", "SYS-1c", "SYS-2", "SYS-9", "SYS-9b", "SYS-10"]):`
```

Comment:
can use the new function `baseline_system_type_compare`

(also line 27)

Reply:
Done